src/hooks/useAuctionBids.ts

```diff
@@ -12,7 +12,7 @@ import { Auction } from 'src/interfaces/Auction'
  * @param auctionId
  */
 export function useAuctionBids(auctionId: string) {
```

Comment:
`useAuctionBids` is going to be removed in a future PR.

Reply:
Okay, then I will revert all these changes after you made a fix. Thank you @adamazad

Reply:
Should I remove this file then? @adamazad